src/currencyviewer/widget/ndeepcurrencyviewer.js

```diff
@@ -23,31 +29,30 @@ mendix.widget.declare("CurrencyViewer.widget.ndeepcurrencyviewer", {
     },

```

Comment:
Both `initContext` and `actRendered`  are deprecated. Just remove. But double check if they are not needed `_Container``